src/styles/index.scss

```diff
@@ -1,3 +1,7 @@
 body {
-  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
```

Comment:
I'd recommend you to reset default margin for `h2` and `p` tags here so you'll be able to remove `margin: 0;` for selectors applied to those tags

Reply:
okey!
I did it!